testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/configurationcache/ConfigurationCacheBuildOperationsFixture.groovy

```diff
@@ -32,6 +34,10 @@ class ConfigurationCacheBuildOperationsFixture {
         this.operations = operations
     }
 
+    boolean getReused() {
+        storeOperation() == null && loadOperation() != null
```

Comment:
❔  In helpers below we use `assertThat(..., nullValue())`. Does one give better experience over the other when there is failure? Should we keep it uniform?

Reply:
`nullValue()` and `notNullValue()` are hamcrest matchers, they only make sense as arguments to something like `assertThat`. This is a simple boolean expression.